Microsoft.Toolkit.Uwp.UI.Controls.DataGrid/DataGrid/DataGrid.cs

```diff
@@ -8999,5 +9068,20 @@ private void VerticalScrollBar_Scroll(object sender, ScrollEventArgs e)
         {
             ProcessVerticalScroll(e.ScrollEventType);
         }
+
+        private void LoadMoreDataFromIncrementalItemsSourceAsync()
```

Comment:
We reserve the Async suffix for methods that return an IAsyncOperation, so this name should be changed to LoadMoreDataFromIncrementalItemsSource().